programming/minibot-assembly-tutorial.md

```diff
@@ -0,0 +1,101 @@
+![Motor Controller Cut Wire](/images/motor-controller-cut-wire.jpg)
+
+2. On the brushed motor speed controller remove the large pcs female connector, from each of the two boards. 
+![Brushed Motor Controller](/images/brushed-motor-controller.jpg)
+
+3. Solder the two ends together (red to red and black to black)
+![Motor Controller Soldered](/images/motor-controlled-soldered.jpg)
+
+4. Take a PCS Female Connector and cut it so it's only an inch, and strip the ends 
+![PCS Female Connector](/images/pcs-female-connector.jpg)
+
+5. Cut a piece of heat shrink in half. Slip one half around the two positive wires and the other around the two negative wires. 
+![Soldered Motor Controller with Heat Shrink](/images/soldered-motor-controller-with-heat-shrink.jpg)
+
+6. Solder the PCS Connector to it's coorisponding color on the motor controllers
+![Soldered to Connector](/images/soldered-to-connector.jpg)
+
+7. Slip the heat shrink over the connection. Use a heat gun or your soldering gun to secure the connection
+![Finished Connection](/images/soldering-done.jpg)
+
+8. You can now connect the motor controller to the two motors through the small pcs connector. 
+
+### POLOLU BOARD:
+1. You will need to solder a small wire within the pololu board, connecting the 5V pin to the first power pin.
+![Motor Controller Back View](/images/motor-controller-back.jpg)
+
+2. Use the collection of 3 female pins on the motor controller, not the one we clipped the power wire off in step 1 of motor speed controllers, to the pololu board. You can connect it to the first column of pins on the board, that are labeled ground, 5V, and signal. It's a bit of a mess, but all the wires coming from the motor controller should be connected to another wire, except for the large pcs connector. 
+![Finished Connection](/images/motor-connections.jpg)
+
+### Battery Supply
+1. You can either use a battery pack made for 6 AA batteries or you can use a 5V rechargable battery pack. You will also need a small pcs connector. 
+![Battery Supplies](/images/battery-pack-unsoldered.jpg)
+
+2. Cut a piece of heat shrink in half, and slip it over the wires coming from the battery pack. '
+![Heat Shrink](/images/heat-shrink.jpg)
+
+3. Solder the battery pack to the pcs connector and add the heat shrink over it. 
+![Completed Battery](/images/battery-done.jpg)
+
+4. You can now screw in your battery pack to the bottom of the minibot and connect it to the last big pcs connector coming from the motor controller.
+
+### Connecting to the RaspberryPi: 
+1. We will be using the raspberry pi to program our minibot. You will use the miniusb to microusb connector to connect the pololu board to the raspberry pi. 
+
+Congratulations, you have finished a minirobot! In our following tutorial we will show you how to write and download code to make your robot move, and how to add different sensors to it! 
```

Comment:
- Are we calling these "minirobots" or "minibots"?
- Can we remove this paragraph until we have another tutorial?